docs/index.adoc

```diff
@@ -4828,36 +4830,71 @@ class Edit {
            // or the name of the editor to use...
            if (" ".equals(info.get("separator"))) { // parameter was not attached to option
 
-               // act as if the user specified --open=defaultEditor
-               args.push(Editor.defaultEditor.name());
+               // act as if the user specified --open=eclipse
+               args.push(editor.name());
```

Comment:
I think it is good to explicitly use the enum constant that represents the default, so `Editor.defaultEditor` (or `Editor.eclipse` if Eclipse were the default).

This is mostly for clarity: so that people reading this code example in the documentation would not have to refer back in the code to what the actual value would be.

I realize that in "real" code there is a tension here with the principle of having information in only once place. To be honest, I am not sure what I would do in a "real" application in this case. :-)

But for documentation purposes, I believe that duplicating the default value in the preprocessor is better because it requires less work on the part of the reader.